docs/backend/behaviors.md

```diff
@@ -11,12 +11,305 @@ myst:
+| plone. textindexer | Full-Text Indexing | Enables the enhanced full-text indexing for a content type. If a field in the schema is marked with the `searchable` directive, its content gets added to the    `SearchableText` index in the catalog |
+| plone.leadimage  | Lead Image | Adds image and image caption fields |
+| plone.locking  | Locking | Locking support for dexterity |
+| plone.translatable  | Multilingual Support | Make this content type multilingual aware. Multilingual support must be installed. |
+| plone.namefromfilename | Name from file name | Automatically generate short URL name for content based on its    primary field file name
+| plone.namefromtitle | Name from title | Automatically generate short URL name for content based on its initial    title
+| plone.navigationroot  | Navigation root | Make all items of this type a navigation root |
+| plone.nextpreviousenabled  | Next previous navigation | Enable next previous navigation for all items of this type |
+| plone.nextprevioustoggle  | Next previous navigation toggle | Allow items to have next previous navigation enabled |
+| plone.ownership  | Ownership | Adds creator, contributor, and rights fields. |
+| plone.relateditems  | Related items | Adds the ability to assign related items |
+| plone.richtext  | RichText | Adds richtext behavior |
+| plone.shortname  | Short name | Gives the ability to rename an item from its edit form. |
+| plone.tableofcontents  | Table of contents | Adds a table of contents |
+| plone.thumb_icon | Thumbs and icon handling | Options to suppress thumbs and/or icons and to override thumb   size in listings, tables etc.
+| plone.versioning  | Versioning | Versioning support with CMFEditions |
+
+## Adding or removing a behavior from a type
+
+There are two ways to add or remove a behavior on a type:
+
+- through the web using the control panel,
+- using a custom add-on GenericSetup profile.
+
+### Through the Web
+
+1. Go to the {guilabel}`Site Setup` and chose the {guilable}`Content Types` control panel
+2. Select the type where you want to add or remove a behavior
+3. Then click on the {guilabel}`Behaviors` tab of the settings of the type.
+4. A list of all available behaviors appears.
+   Select or unselect the checkbox of the behavior you want to add to or remove from the type.
+5. Save the form by clicking on the {guilabel}`Save` button at the bottom of the page.
+
+### Using a GenericSetup profile
+
+Given you already have a custom add-on with a `profiles/default` directory.
+Given you created a custom behavior named `mybehavior.subtitle`.
+
+If you want to enable a behavior on an existing content type, create a new directory `types` under `profiles/default`.
+In there create a file named the same as the content type you want to change.
+In the example here, you want to add a behavior to the built-in Event type.
+The file to create is named `Event.xml` and it is a Factory Type Information definition.
```

Comment:
Please add a term and definition to the Glossary in `glossary.md`. Here's markup for this bit.

```suggestion
The file to create is named `Event.xml` and it is a {term}`Factory Type Information` (FTI) definition.
```